info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -261,18 +274,57 @@ void storeElectionContestInDatabase(String electionQueryId, JsonObject contest)
     datastore.update(electionEntity);
   }
 
-  // @TODO [Find incumbency.]
+  private boolean checkIncumbency(Map<String, String> map, String position, String name) {
+    for (Map.Entry<String, String> e : map.entrySet()) {
+      if (e.getKey() == position && e.getValue() == name) return true;
+    }
+    return false;
+  }
+  private Map<String, String> getincumbents(String division) {
```

Comment:
Do you have unit tests for this?

Reply:
I'm resolving previous comments in this next commit and then I will be writing test in the following commit.